packages/hash/frontend/src/pages/[account-slug]/entities/[entity-uuid].page/entity-editor/links-section/link-table/cells/linked-with-cell/linked-entity-list-editor/max-items-reached.tsx

```diff
@@ -0,0 +1,15 @@
+import { Button } from "@hashintel/hash-design-system";
+
+export const MaxItemsReached = ({ limit }: { limit: number }) => {
+  return (
+    <Button
```

Comment:
Why is this a `Button`, is it meant to be interactible in the future?

Reply:
To achieve the same UI, but in a disabled state.